app/konnect/org-management/oidc-idp.md

```diff
@@ -47,6 +47,60 @@ authentication.
 
 1. Click **Save**.
 
+<!-- copied from okta instructions  as is also relevant to OIDC but needs SME edit for actual details  
+### Map {{site.konnect_short_name}} teams to Okta groups
+By mapping Okta groups to [{{site.konnect_short_name}} teams](/konnect/org-management/teams-and-roles/),
+you can manage a user's {{site.konnect_short_name}} team membership directly through
+Okta group membership.
+
+After mapping is set up:
```

Comment:
@iamnande Are all of the commented out instructions here and below accurate or do they need to be revised?